### PR TITLE
Update cluster TLS certificate process for Globalsign

### PR DESCRIPTION
**Pull request by johnake**
## Context
Following the recent generation of new certificates for TeacherServices.Cloud(which is hosted in AKS) using GlobalSign; we need to document the steps involved for future use

## Changes proposed in this pull request
Update readme within the Cluster TLS section, to include the steps for certificate generation, using GlobalSign

## Checklist

- [x] I have performed a self-review of my code, including formatting and typos
- [x] I have [cleaned the commit history](https://www.annashipman.co.uk/jfdi/good-pull-requests.html)
- [x] I have added the `Devops` label
- [x] I have attached the pull request to the trello card

### CODE REVIEW

**Comment by johnake** [comment]
[Update cluster TLS certificate process for Globalsign](https://trello.com/c/qUbeQDJR/510-update-cluster-tls-certificate-process-for-globalsign)

File: README.md [hunk 1]
```diff
@@ -115,13 +115,53 @@ i.e. "${var.environment}-${var.config}-teacherservices-cloud"
 
 e.g. cluster99-development-teacherservices-cloud
 
+#### GlobalSign certificate generation
+
+##### Generate value for DNS record
```
Comment (saliceti) [inline README.md:120]: These 3 sections should be added to a new section called "Add new top domain"
And mention this has already been done. We don't need to do it again if we add a new cluster.

File: README.md [hunk 2]
```diff
@@ -115,13 +115,53 @@ i.e. "${var.environment}-${var.config}-teacherservices-cloud"
...
+
+1. Select ‘Manage Domains’ in GlobalSign
+2. Search for teacherservices.cloud and select the green check mark
+3. Select verify domain
+4. You should receive a feedback [Your domain has been successfully verified.]
+
+##### Create Certificate in Azure
+
+Pre-requisites:
+Ensure CAA record(for teacherservices.cloud) allows GlobalSign, if not add it, this follows the pattern Flags = 0, Tag = issue, value =“globalsign.com”
```
Comment (saliceti) [inline README.md:149]: Link to the terraform configuration in the repo

File: README.md [hunk 3]
```diff
@@ -115,13 +115,53 @@ i.e. "${var.environment}-${var.config}-teacherservices-cloud"
...
+reference: https://support.globalsign.com/ssl/general-ssl/how-add-dns-caa-record-dns-zone-file
+
+1. Navigate to Key Vaults, select the applicable Key vault 
+2. Either create a new certificate or generate a new version of an existing certificate(the latter is preferred where possible) - Validity is typically left at 12 months
+
```
Comment (saliceti) [inline README.md:154]: Add a step to add the certificate name to the terraform configuration "ingress_cert_name" (link to it)

Comment (saliceti) [reply README.md:152]: Please mention ingress_cert_name on line 140

File: README.md [hunk 4]
```diff
@@ -90,9 +90,44 @@ and then loaded into the cluster using terraform on cluster build.
 
 Initial set up requires manual steps in the cluster Azure KV.
 
-- create KV certificate CA
-    - The CA API key was created in our digicert service account, and is kept in a secret in the prod TSC Domains KeyVault.
-- generate cert within KV certificate
+#### Add new top domain
+
```
Comment (saliceti) [inline README.md:94]: Please add this was required for allowing teacherservices.cloud top domain. It is not needed anymore for new clusters under the same top domain.

**Review by saliceti — APPROVED** [review]

**Pull request merged**